frontend/src/Game.tsx

```diff
@@ -0,0 +1,66 @@
+import React, {useEffect, useState} from 'react';
+import AccountCard from './components/AccountCard';
+
+export interface IAccount {
+    id: number;
+    username: string;
+    img: string;
+    hp: number;
+    xp: number;
+    maxXp: number;
+    level: number;
+    useShaderForProfilePicture: boolean;
+    vertexShaderCode: string;
+    fragmentShaderCode: string;
+}
+
+interface IModalInfo {
+    title: string;
+    description: string;
+    extraHtml: string;
+}
+
+
+const Game: React.FC = () => {
+    const [accounts, setAccounts] = useState<IAccount[]>([]);
+    const [modalInfo, setModalInfo] = useState<IModalInfo | null>(null);
+
+    useEffect(() => {
+        fetch("http://localhost:3000/account").then(res => res.json()).then((res) => {
```

Comment:
Let's make `http://localhost:3000` some config thing from the start to make it easier on us later imo, could be just pull from env var idc tbh

```suggestion
        fetch(`${config.BASE_URL}/account`).then(res => res.json()).then((res) => {
```

Maybe with the format `HEALTYBOICHALLENGE_[varname]`
so
`HEALTYBOICHALLENGE_BASE_URL` or something

or if there's something neat built into react

Reply:
Fixed by using environment variable. By creating a .env file in the root of the frontend folder containing the REACT_APP_BASE_URL="url" React will load it into environment variables making it usable using process.env